apps/api/src/report/report.service.ts

```diff
@@ -439,7 +447,14 @@ export class ReportService {
       this.getAverageTimeWithEachStakeholderGroup(statuses, to).then(report9 => ({ report9 })),
       this.getAverageTimeOfMilestones(to).then(report10 => ({ report10 })),
     ];
-    const report = await Promise.all(promises);
+    const report = await Promise.all(
```

Comment:
is this function used anymore?

Reply:
`/reports/applicant` endpoint uses this function and we can use the endpoint to get all reports at once by postman.